app/models/user.rb

```diff
@@ -0,0 +1,46 @@
+class User < ApplicationRecord
+	
+	
+	has_many :enrollments
+	has_many :courses, through: :enrollments
+	
+	has_many :user_course_progresses
+	
+	
+	#first_name and last_name can't be nul;
+	validates :first_name,:last_name,:email,:phone,:address, presence:true
+	
+	#first_name ans last_name must have at least 3-20 character
+	validates :first_name,:last_name, length: {minimum:3,maximum:20}
```

Comment:
In this case, it will not work. Can I change it to a minimum of two characters, or are there any suggestions for the minimum and maximum character length?